### DISCUSSION TITLE
怎么不增加 [巴拉巴拉，一堆东西] 下载呀？看看隔壁BakaXL。。。

### DISCUSSION THREAD

**Discussion by MacroMeng** [Meloong-Git/PCL#3713] [提问] [answered]
#### 这是BAKA XL
![image](https://github.com/Hex-Dragon/PCL2/assets/132897527/069beb62-1428-4ce7-aca5-47d164962499)

#### 这是HMCL
----------
![image](https://github.com/Hex-Dragon/PCL2/assets/132897527/9d5aa4f8-0807-4b3c-8507-69b2790d2801)

#### 这是PCL
----------
![image](https://github.com/Hex-Dragon/PCL2/assets/132897527/453dde7c-c898-45b8-a158-5c04e9d31aaf)

### 差距咋就这么大呢

**Comment by allMagicNB** [comment]
在投票了.jpg

**Comment by MoYuan-CN** [comment, marked as answer]
有关多个内容的请求，参考下列帖子
- 支持下载资源包 
  - 反馈 #44
  - 投票 #1095
- 支持光影下载 
  - 反馈 #396
  - 投票 #2768
- 支持数据包下载 
  - 反馈 #2991
  - 投票 #2994
- 支持下载地图：存在语言问题
  - https://github.com/Hex-Dragon/PCL2/issues/44#issuecomment-1403698724
  - #135
- 基岩版相关：启动都启动不了更别提下载其相关内容
  - *“这玩意儿我根本启动不了！用官方启动器吧！告辞！”* ——龙腾猫跃
  - #81

**Comment by iPodtouch6557** [comment]
这么好东西你完全可以去用隔壁启动器
我知道你希望PCL更好，同时耐心也是个好东西
